### ISSUE TITLE
Link in readme

### ISSUE THREAD

**Issue by QwEekYhyo** [puceaulytech/drone-project#1]
You changed the path to the reports so the link in the readme doesn't work anymore, please I want it back, it was a cool feature :cry:

**Comment by BlackJackGeary83** [comment]
> You changed the path to the reports so the link in the readme doesn't work anymore, please I want it back, it was a cool feature :cry: 

No, it was decided to make the link unusable because the reports are cringe as Romain is doing absolutely nothing for the project so I thought it would be better to bluff and put a useless link that make people think there are actually useful reports

**Comment by QwEekYhyo** [comment]
Mmmmh, that seems fair enough, it's very unlikely that people are gonna click on the link because let's face it : who cares about reports ? so it is probably best to keep it this way so that people think there is actual content
